http/src/request/channel/webhook/execute_webhook.rs

```diff
@@ -189,6 +191,13 @@ impl<'a> ExecuteWebhook<'a> {
         self
     }
 
+    /// Execute in a thread in the webhook's channel.
```

Comment:
Maybe this should be clarified? Took me a while to figure out what this means. Something about the hierarchy of threads in a channel or something.